src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2921,6 +2991,14 @@ private RefKind GetEffectiveParameterRefKind(ParameterSymbol parameter, RefKind
                     conversions.AddMany(Conversion.Identity, argumentPosition);
                     conversions.Add(conversion);
                 }
+
+                if (badArguments != null && !completeResults)
+                {
+                    // Can this method return after the first bad argument, even in the
```

Comment:
It feels uncomfortable to pose questions like this in the code. I think the open issue is the right place to track the question and its answer. Once/if we learn something useful to the reader, it could be documented here, but this comment just distracts from trying to read the code.